src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationSymbolFactory.cs

```diff
@@ -337,10 +338,11 @@
         bool hasReferenceConstraint = false,
         bool hasUnmanagedConstraint = false,
         bool hasValueConstraint = false,
+        bool allowsRefLikeType = false,
```

Comment:
>bool allowsRefLikeType = false, [](http://example.com/codeflow?start=8&length=31)

Similar comment about order of parameters

Reply:
Especially that in `CodeGenerationTypeParameterSymbol` it comes before `hasUnmanagedConstraint` parameter.